features/steps/steps.py

```diff
@@ -165,17 +165,16 @@ def step_impl(context, attribute, value):
     )
 
 
-@given('A file with {field} "{value}"')
-def step_impl(context, field, value):
+@given('A file with {field} "{values}"')
+def step_impl(context, field, values):
+    values = list(values.replace('"', "").split(', '))
     if field == "Model View Definition":
-        applicable = get_mvd(context.model) == value
+        applicable = any([get_mvd(context.model) == value for value in values])
     elif field == "Schema Identifier":
-        applicable = context.model.schema.lower() == value.lower()
+        applicable = any([context.model.schema.lower() == value.lower() for value in values])
```

Comment:
With values already containing all lowercase (due to comment above) you could then simply do

applicable = context.model.schema.lower() in values

Reply:
The works for Schema Identifier, but not Model View Definition. The line you wrote converts a string to lower cases, but then the value for model view definition will not match 'get_mvd(context.model)' in some IFC2X3 schemas in other rules. An option would be to convert this to lower cases as well, but this throws an error since the get_mvd function has an error handler to return None (TypeError). I've changed the except condition in get_mvd() function to return an empty string instead, and for the function to return lower cases. 

The changes are in, to make it clearer https://github.com/buildingSMART/ifc-gherkin-rules/pull/15/commits/4b6c110e75f4be3bda8d04836ac1ae696336cd41 
Maybe you can take a quick look at it @aothms ? Another option would be to have a workaround in the function linked to the given statement (making sure get_mvd does not return None), which one has your preference?
Pytest runs well in both options

Reply:
@Ghesselink https://peps.python.org/pep-0505/ would have made that easier "none-aware operators", but it looks like it's not going to make it.

I prefer to have get_mvd() unaltered to keep this kind of detail as local as possible, we might later need to print it to the user for example.

Most concise would be:

`(get_mvd() or '').lower() in ...`

It would also not propagate the None. So maybe best is something like:

```
conditional_lowercase = lambda s: s.lower() if s else None
conditional_lowercase(get_mvd()) in ...
```

Reply:
Doesn't look like it, created in 2015 and still deferred .. In the discussion programming gets almost a spiritual/philosophical component, 'is there something when there's nothing'? ;p Maybe not a coincidence that quite a bit of philosophy students turned to programming. 

Anyway, what you suggested is tested and works fine in https://github.com/buildingSMART/ifc-gherkin-rules/pull/15/commits/b9c8689101207aa3f68d66c8b46e11274e9f0b66